Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
```

Comment:
Best practice is to specify access modifiers explicitly (make it public, right now it's internal). You don't need to know what internal means yet.